src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -121,14 +116,22 @@ ApplicationManagerImpl::ApplicationManagerImpl()
                                       &ApplicationManagerImpl::OnTimerSendTTSGlobalProperties,
                                       true),
     is_low_voltage_(false) {
+
+    devicesType.insert(
```

Comment:
May be create method, which return this map.
It will allow to make *deviceType* as a const and help to avoid modification problems.